test/unit/test_config.py

```diff
@@ -93,18 +85,17 @@ def test_config_copy():
     assert c1 != mod_copy_c1
 
 
-class ConfigWithMissingAttributes(config.Config):
-    def __init__(self, existing_attribute, new_attribute="new_attribute"):
-        super().__init__()
-        self.existing_attribute = existing_attribute
-        self.new_attribute = new_attribute
+@dataclass
+class ConfigWithMissingAttributes(Config):
+    existing_attribute: str
+    new_attribute: str = "new_attribute"
 
 
 def test_config_missing_attributes_filled_with_default():
     # when we load a configuration object that does not contain all attributes as the current version of the
     # configuration object we expect the missing attributes to be filled with the default values taken from the
     # __init__ method.
 
-    config_obj = config.Config.load("test/data/config_with_missing_attributes.yaml")
+    config_obj = Config.load("test/data/config_with_missing_attributes.yaml")
```

Comment:
The comment above isn't correct anymore (refers to the `__init__` method).

Reply:
changed